packages/grafana-ui/src/components/PageLayout/PageToolbar.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { PageToolbar } from '..';
+
+describe('PageToolbar', () => {
+  it('renders left items when title is not set', () => {
+    const testId = 'left-item';
+    render(
+      <PageToolbar
+        leftItems={[
+          <div key="left-item" data-testid={testId}>
+            Left Item!
+          </div>,
+        ]}
+      />
+    );
+
+    expect(screen.getByTestId(testId)).toBeInTheDocument();
```

Comment:
Its nitpicking, but i think it would be better if it tested without the test ID

```
    expect(screen.getByTextContent("Left Item!")).toBeInTheDocument();
```

Reply:
yep, makes sense!